app-text/sword/sword-1.8.1-r1.ebuild

```diff
@@ -42,12 +42,12 @@ src_configure() {
 	local mycmakeargs=(
 		-DSYSCONF_INSTALL_DIR="${EPREFIX}/etc"
 		-DLIB_INSTALL_DIR="${EPREFIX}/usr/$(get_libdir)"
+		-DLIBSWORD_LIBRARY_TYPE=$(usex static-libs Static Shared)
```

Comment:
I am pretty sure this will result in USE=static-libs causing the shared library not to be built at all. This is better than the old behaviour (in which case USE=static-libs does the same but USE=-static-libs installs both the static and the shared library) but still wrong.

Anyway - please remove this commit from the PR, @jsmolic . Apart from the fact this has got nothing to do with EAPI-7, in my opinion such a change goes beyond the scope of what is allowed in stabilised ebuilds - and app-text/sword-1.8.1-r1 is stable on both amd64 and x86.

Reply:
:+1: